smoke-test/src/test/java/org/opennms/smoketest/IntegrationAPIIT.java

```diff
@@ -99,9 +99,9 @@ private static void verifyHealthCheckWithDescription(final InetSocketAddress ssh
                         for (String line : shellOutput.split("\\r?\\n")) {
                             if (line.contains(healthCheckDescription)) {
                                 LOG.info("Health check result: {}", line);
-                            }
-                            if (line.contains("Success")) {
-                                healthCheckSuccess = true;
+                                if (line.contains("Success")) {
```

Comment:
If we want to keep this change here, we'll need to push a new branch for this that ends in `-smoke` and verify that this test actually passes with the change.